x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/scalar/string/Wildcard.java

```diff
@@ -31,7 +31,7 @@
 
 /**
  * EQL wildcard function. Matches the form:
- *     wildcard(field, "*wildcard*pattern*", ...)
+ *     field like ("*wildcard*pattern*", ...)
  */
 public class Wildcard extends BaseSurrogateFunction {
```

Comment:
No need to keep this class around.
It's used inside the parser `ExpressionBuilder#visitOperatorExpressionDefault` however its only role is to call `makeSubstitute` which is similar to `ExpressionBuilder#combineExpressions`. See the handling of `EqlBaseParser.IN_INSENSITIVE:` in `visitOperatorExpressionDefault`.

Reply:
Thanks. Class removed.